integration-tests/typescript-fetch/src/generated/todo-lists.yaml/models.ts

```diff
@@ -12,7 +12,7 @@ export type t_CreateUpdateTodoList = {
 }
 
 export type t_TodoList = {
-  created: string
+  created: Date
```

Comment:
This will be problematic without `--enable-runtime-response-validation` enabled, as by default we don't use runtime parsing in the client templates